CHANGELOG.md

```diff
@@ -35,6 +35,15 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - Don't keep volume open after acquiring a read-storage failed in migrate/copy/virtual full [PR #1106]
 - webui: show DIR message if ACL prevents a job rerun [PR #1110]
 
+### Added
+- Python plugins: add default module_path to search path [PR #1038]
+- tests: simplify test coverage analysis [PR #1010]
+- docs: Add chapter for mariabackup db plugin [PR #1016]
+- dird: extend the list command to be able to query volumes and pools by ID [PR #1041]
+- systemtests: wait for mariadb shutdown [PR #1048]
+- webui: analytics module: show stored data per jobname in treemap [PR #1082]
```

Comment:
Isn't this line and the above too  much additions for CHANGELOG.md? I would expect one single line.

Reply:
Yes, this got messed up a bit due to having two added sections previously. I've corrected it.